django/contrib/admin/templates/admin/pagination.html

```diff
@@ -1,6 +1,6 @@
 {% load admin_list %}
 {% load i18n %}
-<p class="paginator">
+<div class="paginator">
```

Comment:
Not sure if this is necessary or not. It could probably be removed. But it's not a paragraph or phrasing content so it seemed reasonable to make this semantic and remove any default styling while I was looking around.

Reply:
I wouldn't change this (just yet?) because there are other templates using the same layout with `<p>` (tests/admin_views/templates/admin/admin_views/article/pagination.html, django/contrib/admin/templates/admin/object_history.html), so changing in one place and not the others feels incomplete.